pkg/generate/docker_gen.go

```diff
@@ -0,0 +1,121 @@
+package generate
+
+import (
+	"errors"
+	"os"
+	"regexp"
+	"strconv"
+	"strings"
+
+	"github.com/moby/buildkit/frontend/dockerfile/parser"
+	"github.com/ocibuilder/ocibuilder/pkg/apis/ocibuilder/v1alpha1"
+	"github.com/sirupsen/logrus"
+)
+
+type DockerGenerator struct {
+	Filepath  string
+	ImageName string
+	Tag       string
+	Logger    *logrus.Logger
+}
+
+func (d DockerGenerator) Generate() ([]byte, error) {
```

Comment:
I know that idiomatic golang allows for these types of variable declaration.  I am not a fan of the single letter though.  feel it negatively impacts readability instead of the inverse

Reply:
Yeah I'm inclined to agree, I think a cleanup is in order for our structs to rename to something more readable. I'll create an issue.